lambdas/check-coi/src/main/java/uk/gov/di/ipv/core/checkcoi/CheckCoiHandler.java

```diff
@@ -98,26 +113,54 @@ public Map<String, Object> handleRequest(ProcessRequest request, Context context
                             .toList();
 
             var coiSubjourneyType = ipvSession.getCoiSubjourneyType();
-            var successfulCheck =
+            var coiCheck =
                     switch (coiSubjourneyType) {
-                        case GIVEN_NAMES_ONLY, GIVEN_NAMES_AND_ADDRESS -> userIdentityService
-                                .areFamilyNameAndDobCorrelatedForCoiCheck(credentials);
-                        case FAMILY_NAME_ONLY, FAMILY_NAME_AND_ADDRESS -> userIdentityService
-                                .areGivenNamesAndDobCorrelated(credentials);
-                        case ADDRESS_ONLY -> userIdentityService.areVcsCorrelated(credentials);
+                        case GIVEN_NAMES_ONLY, GIVEN_NAMES_AND_ADDRESS -> new CoiCheck(
+                                userIdentityService.areFamilyNameAndDobCorrelatedForCoiCheck(
+                                        credentials),
+                                CoiCheckType.LAST_NAME_AND_DOB);
+                        case FAMILY_NAME_ONLY, FAMILY_NAME_AND_ADDRESS -> new CoiCheck(
+                                userIdentityService.areGivenNamesAndDobCorrelatedForCoiCheck(
```

Comment:
More consistent method naming with the other method (ending in "ForCoiCheck": they both are for the coi check)